download.html

```diff
@@ -98,8 +98,8 @@ <h3>Latest stable release: {{versionFull}} ({{date}})</h3>
 		Other packages and versions are available <a href="#versionList">below</a>. For an overview of the current maintenance state of the different OpenUI5 versions, please check the <a href="https://openui5.hana.ondemand.com/versionoverview.html">version overview page</a>.
 
 		<div id="noUnstableVersion" style="display: none">
-			<h3>Latest unstable development build:</h3>
-			Currently there is no newer preview release available. It will be added, as {{versionNext}} development progresses.
+			<h3>Unstable Releases - Change in Release Strategy</h3>
```

Comment:
i would reword to "unstable development builds" like on the left side - the term unstable release is not well-accepted in our org

Reply:
done